### PR TITLE
create Footer for PPP with some basic building blocks

### PR DESCRIPTION
**Pull request by kilbergr**
This footer includes styling from USDS https://standards.usa.gov/page-templates/#documentation-page for buttons, footer, contact info, and place for logo.
Should abide by conventions as detailed on the USA.gov site standards page. First foray into using jsx so please don't hesitate to criticize!

### CODE REVIEW

File: client/src/shared/Footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        </div>
+      </div>
+
+      <div class="usa-footer-secondary_section">
+        <div class="usa-grid">
+          <div class="usa-footer-logo usa-width-one-half">
+            <a href="https://www.ustranscom.mil/">
+              <img
+                class="usa-footer-logo-img"
+                src=""
```
Comment (ntwyman) [inline client/src/shared/Footer/Footer.jsx]: img needs some src
screen shot 2017-12-28 at 2 20 37 pm

File: client/src/shared/Footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+function Footer() {
+  return (
+    <footer class="usa-footer usa-footer-medium" role="contentinfo">
+      <div class="usa-grid usa-footer-return-to-top">
+        <a href="">Return to top</a>
+      </div>
+      <div class="usa-footer-primary-section">
```
Comment (ntwyman) [inline client/src/shared/Footer/Footer.jsx]: Within a react component you should use className in place of class.  [This site](https://magic.reactjs.net/htmltojsx.htm) is a useful tool if you are cutting and pasting HTML.

File: client/src/shared/Footer/Footer.jsx (hunk 1 above)
Comment (ntwyman) [reply client/src/shared/Footer/Footer.jsx]: You should be able to find an example from move.mil and add it to our site (it's an svg file)

**Comment by tenshiemi** [comment]
Also need to rebase this on master!

File: client/src/shared/Footer/Footer.jsx (hunk 1 above)
Comment (kilbergr) [reply client/src/shared/Footer/Footer.jsx]: OK.

**Pull request merged**